reference/7.4/Microsoft.PowerShell.Core/About/about_Scopes.md

```diff
@@ -96,6 +83,34 @@ To get all the variables in the global scope, type:
 Get-Variable -Scope global
 ```
 
+A child scope doesn't inherit the variables, aliases, and functions from the
```

Comment:
this is a confusing paragraph. I think it's the "inherit" term. When a reference is made to a variable, alias, or function, the current scope will be search, then if not found, the parent scope is search (in a chain all the way to global). _If_ a variable is _private_ in a _parent_ scope, the search through the scopes will not resolve and the search will continue through the scope chain.

here's an example:

```powershell
PS> cat /tmp/scop.ps1
function A {
	"Setting `$funcAVar1 to 'i am function a'"
	$funcAVar1 = "i am function a"
	B
}

function B {
	"in B before set -> $funcAVar1"
	$private:funcAVar1 = "I've locally overwritten it - child scopes can't see me!"
	"in B after set  -> $funcAVar1"
	C
}

function C {
	"in C before set -> $funcAVar1 - should be the value set in function A"
	$funcAVar1 = "my child scope will see this change"
	"in C after set  -> $funcAVar1"
	D
}

function D {
	"in D -> $funcAVar1 - should be the value set in function C"
}

A
PS> /tmp/scop.ps1
Setting $funcAVar1 to 'i am function a'
in B before set -> i am function a
in B after set  -> I've locally overwritten it - child scopes can't see me!
in C before set -> i am function a - should be the value set in function A
in C after set  -> my child scope will see this change
in D -> my child scope will see this change - should be the value set in function C
PS>
```